examples/ci/ci.tf

```diff
@@ -14,12 +14,21 @@ provider "aws" {
 }
 
 module "acs" {
-  source = "github.com/byu-oit/terraform-aws-acs-info?ref=v3.5.0"
+  source = "github.com/byu-oit/terraform-aws-acs-info?ref=v4.0.0"
+}
+
+// test existing cluster
+resource "aws_ecs_cluster" "existing" {
+  name = "test"
 }
 
 module "fargate_api" {
-  source         = "../../"
-  app_name       = "example-api"
+  source   = "../../"
+  app_name = "example-api"
+  existing_ecs_cluster = {
+    arn = aws_ecs_cluster.existing.arn
+    id  = aws_ecs_cluster.existing.id
+  }
```

Comment:
Maybe move these (example of how to use an existing cluster) to the simple example instead of the ci file?